.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
Let's not add text editor specific ignores, because there's a near endless quantity of text editors. You should just set a `gitignore` file, pointed to by `core.excludesFile` in your global/system gitconfig, containing things for whatever text editor you so desire, rather than relying on every git repository you touch having such ignores set.